src/Simulation/QCTraceSimulator/DepthCounter.cs

```diff
@@ -46,6 +55,16 @@ public DepthCounter(IDoubleStatistic[] statisticsToCollect = null)
             OperationCallRecord opRec = new OperationCallRecord();
             opRec.OperationName = CallGraphEdge.CallGraphRootHashed;
             operationCallStack.Push(opRec);
+
+            this.optimizeDepth = optimizeDepth;
+            if (optimizeDepth) {
```

Comment:
I believe, our C# style guide requires to place braces on separate lines. All of the new code in this file is affected.

Reply:
Yes, I will address this later.

Reply:
Updated.